core/vmutils/src/main/java/org/arakhne/afc/vmutil/ThreadServiceFinder.java

```diff
@@ -38,7 +39,7 @@
  */
 public final class ThreadServiceFinder {
 
-	private static final Stack<ThreadServiceProvider> SERVICES = new Stack<>();
+	private static final Deque<ThreadServiceProvider> SERVICES = new ArrayDeque<>();
```

Comment:
This replacement of Stack by Deque may cause invalid behavior in multi-threaded applications. It will be ignored in the merged.